features/step_definitions/common_steps.rb

```diff
@@ -316,6 +316,16 @@
   edit_link.click
 end
 
+When /I click the summary table '(.*)' (link|button) for the '(.*)' link$/ do |link_name, elem_type, field_to_edit|
+  case elem_type
+    when 'link'
+      edit_link = page.find(:xpath, "//tr/td/*/a[normalize-space(text()) = '#{field_to_edit}']/../../..//a[contains(normalize-space(text()), '#{link_name}')]")
+    else
+      edit_link = page.find(:xpath, "//tr/td/*/a[normalize-space(text()) = '#{field_to_edit}']/../../..//input[normalize-space(@value) = '#{link_name}']")
```

Comment:
If you use `string()` instead of `text()` here it'll not get tripped up when used against something that has had e.g. a `<strong>` slipped into it.